.github/workflows/build.yml

```
@@ -186,6 +189,9 @@ jobs:
           
           ELASTICSEARCH_HOST=${{ secrets.ELASTICSEARCH_HOST }}
           
+          RESEND_API_KEY: ${{ secrets.RESEND_API_KEY }}
+          RESEND_FROM_EMAIL: ${{ secrets.RESEND_FROM_EMAIL }}
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)